### PR TITLE
Registration email sent template with RA locations or RAs

### PR DESCRIPTION
**Pull request by arothuis**
* Adds specific templates for RA locations or RAs for when a registration email has been sent, replacing the old template for RAs
* Adds translations for these templates

### CODE REVIEW

**Comment by arothuis** [comment]
@DRvanR, a first draft

File: app/Resources/translations/messages.en_GB.xliff [hunk 1]
```diff
@@ -689,7 +707,7 @@ An e-mail with your activation code has been sent to the e-mail address %email%.
       <trans-unit id="d3810fbfb6254169454c8c68850947acce991691" resname="subscriberNumber">
         <jms:reference-file line="41">Form/Type/SendSmsChallengeType.php</jms:reference-file>
         <source>subscriberNumber</source>
-        <target state="new">subscriberNumber</target>
+        <target>subscriberNumber</target>
```
Comment (DRvanR) [inline app/Resources/translations/messages.en_GB.xliff]: ?

Comment (arothuis) [inline app/Resources/translations/messages.en_GB.xliff]: "?" indeed. It should be [ignored][0] for translations. I think I accidentally touched it in the JMS Translation UI, so it changed state.

[0]: https://github.com/SURFnet/Stepup-SelfService/blob/develop/src/Surfnet/StepupSelfService/SelfServiceBundle/Form/Type/SendSmsChallengeType.php#L41

**Comment by arothuis** [comment]
Some improvements, @DRvanR

**Comment by DRvanR** [comment]
:+1:

**Pull request merged**